app.js

```diff
@@ -8,4 +58,86 @@ hamburguer.addEventListener('click', () => {
+                <div class="content_card">
+                    <h2 class="title_card">${work.title}</h2>
+                    <div class="subtitle_card">
+                        <h6 class="first_subtitle_card">${work.subtitle1}</h6>
+                        <img class="point_icon" src="images/point_icon.png" alt="point">
+                        <h6 class="second_subtitle_card">${work.subtitle2}</h6>
+                        <img class="point_icon" src="images/point_icon.png" alt="point">
+                        <h6 class="second_subtitle_card">${work.year}</h6>
+                    </div>
+                    <p class="paragraph_card">
+                        ${work.paragraph}
+                    </p>
+                    <div>
+                        <ul class="technologies">
+                            <li>${work.technologies[0]}</li>
+                            <li>${work.technologies[1]}</li>
+                            <li>${work.technologies[2]}</li>
+                        </ul>
+                    </div>
+                    <button onClick="popupModal(${work.id})" type="button" class="button_card button"><b>See project</b></button>
+                </div>
+            </div>
+
+   `;
+});
+
+function popupModal(x) {
+  modal.classList.toggle('show');
+
+  modal.innerHTML += `
+                <div class="modal">
+                    <div class="modal_content">
+                        <div class="header_modal">
+                            <h2 class="title_modal">${projects[x].title}</h2>
+                            <img class="modal_close" onClick="closeModal()" src="images/close_icon.png" alt="close">
+                        </div>
+                        <div class="subtitle_card">
+                            <h6 class="first_subtitle_card">${projects[x].subtitle1}</h6>
+                            <img class="point_icon" src="images/point_icon.png" alt="point">
+                            <h6 class="second_subtitle_card">${projects[x].subtitle2}</h6>
+                            <img class="point_icon" src="images/point_icon.png" alt="point">
+                            <h6 class="second_subtitle_card">${projects[x].year}</h6>
+                        </div>
+                        <div class="content_image_modal">
+                            <img class="image_modal" src="${projects[x].image}" alt="snapshot tonic">
+                        </div>
+                        <div class="modal_footer">
+                        <p class="paragraph_card paragraph_modal">
+                            ${projects[x].paragraph}
+                        </p>
+                        <div class="right_modal">
+                            <ul class="technologies">
+                                <li>${projects[x].technologies[0]}</li>
+                                <li>${projects[x].technologies[1]}</li>
+                                <li>${projects[x].technologies[2]}</li>
+                            </ul>
```

Comment:
- [ ] Please you can use the `map` function in JavaScript to populate the `<ul>` element with the technologies from an array. :pray: